doc/source/whatsnew/v2.0.0.rst

```diff
@@ -125,6 +125,10 @@ Copy-on-Write improvements
   a modification to the data happens) when constructing a Series from an existing
   Series with the default of ``copy=False`` (:issue:`50471`)
 
+- The :class:`DataFrame` constructor, when constructing a DataFrame from a dictionary
+  of Series objects and specifying ``copy=False``, will now use_inf_as_null a lazy copy
```

Comment:
> use_inf_as_null

This seems to be a copy and paste error?

Reply:
I suppose an automatic completion of my editor when just wanting to write "use" .. ;)